fastlane_core/lib/fastlane_core/ui/implementations/shell.rb

```diff
@@ -150,6 +150,21 @@ def password(message)
 
     private
 
+    def encode_as_utf_8_if_possible(message)
+      return message if message.valid_encoding?
+
+      # Try following common encodings and return the first with valid codings
+      #
+      # * UTF-16 - output by genstrings
+      test_string = message.dup
+      [Encoding::UTF_16].each do |dest_encoding|
```

Comment:
Is this needed when there's only one encoding to be tested on?

Reply:
Yeah, that's why I asked of keeping the array now that we only have one left, not that it'll cost much computationally talking😂